Makefile

```diff
@@ -1,25 +1,204 @@
-all: .restart-cqf-ruler .update-cqf-ruler-image
+all r4: .setup-cqf-ruler-r4 synthea generate-patients-r4 calculate-patients-r4
 
 info:
 	$(info usage: `make MEASURE_DIR=/path/to/measure/dir VERSION=x.y.z)
 
-.seed-cqf-ruler-no-vs:
-	make -C $(MEASURE_DIR) connectathon .seed-cqf-ruler-no-vs
-	touch .seed-cqf-ruler-no-vs
+stu3: .setup-cqf-ruler-stu3 synthea generate-patients-stu3 calculate-patients-stu3
 
-.run-load-script:
-	until `curl --output /dev/null --silent --head --fail http://localhost:8080/cqf-ruler-dstu3`; do printf '.'; sleep 5; done
-	./load-cqf-ruler.sh $(MEASURE_DIR)
+.setup-cqf-ruler-stu3: .new-cqf-ruler connectathon .seed-measures-stu3 .seed-vs-stu3
+	touch .setup-cqf-ruler-stu3
 
-.restart-cqf-ruler:
-	-docker stop cqf-ruler
+.setup-cqf-ruler-r4: .new-cqf-ruler connectathon .seed-measures-r4 .seed-vs-r4
+	touch .setup-cqf-ruler-r4
+
+.new-cqf-ruler:
 	docker run --name cqf-ruler --rm -dit -p 8080:8080 contentgroup/cqf-ruler:develop
+	touch .new-cqf-ruler
+
+connectathon:
+	$(info connectathon checks out a specific commit SHA in case filepaths are updated)
+	git clone https://github.com/DBCG/connectathon.git
+	cd connectathon && git checkout 52084217d33a9d9fc8d79664a535edb24557635b
+
+.wait-cqf-ruler:
+	until `curl --output /dev/null --silent --head --fail http://localhost:8080/cqf-ruler-r4`; do printf '.'; sleep 5; done
+
+.seed-measures-r4: .wait-cqf-ruler
+	# CMS 104
+	curl -X PUT http://localhost:8080/cqf-ruler-r4/fhir/Measure/measure-EXM104-FHIR4-8.1.000 \
+		-H 'Content-Type: application/json' \
+		-d @connectathon/fhir4/bundles/EXM104_FHIR4-8.1.000/EXM104_FHIR4-8.1.000-files/measure-EXM104_FHIR4-8.1.000.json
+	curl -X PUT http://localhost:8080/cqf-ruler-r4/fhir/Library/library-EXM104-FHIR4-8.1.000 \
+		-H 'Content-Type: application/json' \
+		-d @connectathon/fhir4/bundles/EXM104_FHIR4-8.1.000/EXM104_FHIR4-8.1.000-files/library-EXM104_FHIR4-8.1.000.json
+	curl -X POST http://localhost:8080/cqf-ruler-r4/fhir \
+		-H 'Content-Type: application/json' \
+		-d @connectathon/fhir4/bundles/EXM105_FHIR4-8.1.000/EXM105_FHIR4-8.1.000-files/library-deps-EXM105_FHIR4-8.1.000-bundle.json
+	# CMS 124
+	curl -X PUT http://localhost:8080/cqf-ruler-r4/fhir/Measure/measure-EXM124-FHIR4-8.2.000 \
+		-H 'Content-Type: application/json' \
+		-d @connectathon/fhir4/bundles/EXM124_FHIR4-8.2.000/EXM124_FHIR4-8.2.000-files/measure-EXM124_FHIR4-8.2.000.json
+	curl -X POST http://localhost:8080/cqf-ruler-r4/fhir \
+		-H 'Content-Type: application/json' \
+		-d @connectathon/fhir4/bundles/EXM124_FHIR4-8.2.000/EXM124_FHIR4-8.2.000-files/library-deps-EXM124_FHIR4-8.2.000-bundle.json
+	curl -X PUT http://localhost:8080/cqf-ruler-r4/fhir/Library/library-EXM124-FHIR4-8.2.000 \
+  		-H 'Content-Type: application/json' \
+		-d @./connectathon/fhir4/bundles/EXM124_FHIR4-8.2.000/EXM124_FHIR4-8.2.000-files/library-EXM124_FHIR4-8.2.000.json
+	# CMS 125
+	curl -X PUT http://localhost:8080/cqf-ruler-r4/fhir/Measure/measure-EXM125_FHIR4-7.2.000 \
```

Comment:
```suggestion
	curl -X PUT http://localhost:8080/cqf-ruler-r4/fhir/Measure/measure-EXM125-FHIR4-7.2.000 \
```

Reply:
Huh I definitely combed though and copy-pasted all the IDs for every measure and library, but I guess that somehow didn't get committed. Anyway, they were all wrong not just this one so I just pushed a fix.

Reply:
Unfortunately the ids in the http requests should have hyphens, but the filenames have underscores.  Your commit changed the filenames, so they are wrong now.  I think the one I suggested was the only {id, filename} that was incorrect.

Reply:
🤦‍♂ welp that explains the mysterious "loss" of my changes